.circleci/config.yml

```diff
@@ -1,5 +1,21 @@
 version: 2
 jobs:
+  build:
+    docker:
+      - image: circleci/node:latest
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run: npm install
+      - run: 
+          name: Docker compose up
+          command: |
+            set -ex
+            cp ./services/registry/.env-example ./services/registry/.env
+            cp ./services/web/.env-example ./services/web/.env
+            cp ./services/storage/.env-example ./services/storage/.env
+            docker-compose up -d
```

Comment:
I threw in a short sleep before the `curl` to give the containers time, and that worked on my build. I think 10 seconds would be plenty.